distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -854,9 +854,7 @@ type Table
             _ : Date_Range -> Error.throw (Unsupported_Database_Operation.Error "Cannot use `Date_Range` for `set` in the database.")
             _ -> Error.throw (Illegal_Argument.Error "Unsupported type for `Table.set`.")
 
-        name_is_generated column = case column of
-            _ : Column -> False
-            _ -> True
+        name_is_generated column = column.is_a Column . not
```

Comment:
Shall we actually move this into the `new_column_name` block below? It could be simpler that way.

Reply:
Done